src/itest/java/com/iexec/sms/tee/GramineBeansLoadingTests.java

```diff
@@ -11,7 +11,7 @@
 import org.springframework.test.context.ActiveProfiles;
 
 
-@ActiveProfiles(profiles = "gramine")
+@ActiveProfiles(profiles = {"gramine", "test"})
```

Comment:
Why not adding `"test"` to `TeeBeansLoadingTests`?

Reply:
Good catch and I tried it. Sadly, this does not work.
Following the inheritance, profiles are appended and the last declared profile becomes kind of the main profile.
If "test" profile is applied on `TeeBeansLoadingTests`, the profiles list becomes ["mock-chain", "test", "gramine"]
In this case (and I don't know why), the `src/main/resource/application-gramine.yml` is being loaded and shadows the `src/itest/resources/application-test.yml` and tests are broken.
To stay efficient, I settled to the current implementation

Reply:
Ok :heavy_check_mark: